lib/views/setting/setting_view.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:clnapp/utils/app_provider.dart';
+import 'package:flutter/material.dart';
+import 'package:file_picker/file_picker.dart';
+
+class SettingView extends StatefulWidget {
+  final AppProvider provider;
+
+  const SettingView({Key? key, required this.provider}) : super(key: key);
+
+  @override
+  State<SettingView> createState() => _SettingViewState();
+}
+
+class _SettingViewState extends State<SettingView> {
+  String? nickName;
+
+  String connectionClient = 'gRPC connection';
+
+  String selectedPath = "No path found";
+
+  String host = "localhost";
```

Comment:
```suggestion
    Setting setting = Setting();
```

Reply:
Ya I have already made this change!! thanks